recipes/newmat/all/test_v1_package/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.1)
```

Comment:
Please, remove the test_v1_package folder. For more info, read https://github.com/conan-io/conan-center-index/discussions/17406